### PR TITLE
[admin] (Links) Removing links to old Excel open spec

### PR DESCRIPTION
**Pull request by AlexJerabek**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ba0d874](https://github.com/OfficeDev/office-js-docs-pr/pull/970/commits/ba0d8749371c3de06dc82fd025cb4d869750a66a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/openspec.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJerabek-patch-1/docs/reference/openspec.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905061638347713-970%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ai%2BjzKicySouHX0QsvkPXYf7WFrQyPTIrh0gZw7dFaU%3D&st=2019-05-06T16%3A34%3A52Z&se=2019-06-06T16%3A39%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/openspec.md [hunk 1]
```diff
@@ -7,16 +7,16 @@ localization_priority: Priority
...
+The Office JavaScript API open specifications provide information about new JavaScript APIs that are being designed for Excel, Word, and other host applications.
 
 > [!IMPORTANT]
-> Features described in the API open specifications may be in various stages of development, such as early design or public preview, and are subject to change. When an API feature becomes generally available, related documentation is removed from the open specification and added to the [Office Add-ins documentation](/office/dev/add-ins/). 
+> Features described in the API open specifications may be in various stages of development, such as early design or public preview, and are subject to change. When an API feature becomes generally available, the [reference documentation](/javascript/api/overview) will be updated.
```
Comment (Reezaali) [inline docs/reference/openspec.md]: Broken link?

File: docs/reference/openspec.md [hunk 2]
```diff
@@ -7,16 +7,16 @@ localization_priority: Priority
...
 > [!IMPORTANT]
-> Features described in the API open specifications may be in various stages of development, such as early design or public preview, and are subject to change. When an API feature becomes generally available, related documentation is removed from the open specification and added to the [Office Add-ins documentation](/office/dev/add-ins/). 
+> Features described in the API open specifications may be in various stages of development, such as early design or public preview, and are subject to change. When an API feature becomes generally available, the [reference documentation](/javascript/api/overview) will be updated. 
 
 ## New Excel JavaScript APIs
 
 Join us in reviewing our design for new Excel JavaScript APIs. 
 
-**See the [Excel JavaScript open specification page](https://github.com/OfficeDev/office-js-docs/tree/ExcelJs_OpenSpec) to learn more and provide your feedback.**
+**See the [Excel JavaScript preview APIs](/javascript/api/excel) to learn more and provide your feedback.**
```
Comment (Reezaali) [inline docs/reference/openspec.md]: Broken link?

File: docs/reference/openspec.md [hunk 3]
```diff
@@ -7,16 +7,16 @@ localization_priority: Priority
 
 # API open specifications
 
-The Office JavaScript API open specifications provide information about new JavaScript APIs that are being designed for Excel, Word, and other host applications. You can review the open specifications in [branches](https://github.com/OfficeDev/office-js-docs/branches/all) of the **OfficeDev/office-js-docs** GitHub repository to learn about APIs that are planned for future releases and can provide feedback about the planned API features and design by logging issues in the GitHub repository.
+The Office JavaScript API open specifications provide information about new JavaScript APIs that are being designed for Excel, Word, and other host applications.
```
Comment (Reezaali) [inline docs/reference/openspec.md]: Can we link to the What's new page where someone can get the latest info on the Preview Req Set? https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4fd2e6c](https://github.com/OfficeDev/office-js-docs-pr/pull/970/commits/4fd2e6cd01b952b6aca639bff524a4c99785504f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/openspec.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJerabek-patch-1/docs/reference/openspec.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905061709216260-970%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=n%2FncVBEuOMTW%2B1H48QqZgh9LOFxeI89A8glgp8XTaRo%3D&st=2019-05-06T17%3A05%3A32Z&se=2019-06-06T17%3A10%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d78a802](https://github.com/OfficeDev/office-js-docs-pr/pull/970/commits/d78a80298b8eb6293ac1d03d32fd9e4cd85286d5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/openspec.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJerabek-patch-1/docs/reference/openspec.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905061711313815-970%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=XgvXZL2Oh3YgvRKu8EHRwWhMc2q7w%2BmD%2BQZzW2z%2Fb1g%3D&st=2019-05-06T17%3A07%3A41Z&se=2019-06-06T17%3A12%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Thanks @Reezaali. The links will have issues because they're relative. Please try them on the [staging branch](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/openspec?branch=AlexJerabek-patch-1). Note that links that go cross-repo will try to go to the patch branch, instead of master.

**Comment by sumurthy** [comment]
Looks good to me.

**Pull request merged**